agent/result_parser.py

```diff
@@ -31,10 +31,9 @@ def parse_results(results: whois.parser.WhoisCom) -> Iterator[Dict[str, Any]]:
         if name is not None:
             names.add(name.lower())
 
-    contact_name = scan_output_dict.pop("name", "")
     for name in names:
         if name != "":
-            output = {
+            output: dict[str, Any] = {
```

Comment:
I don't think it is any it is either a list or string from the code below

Reply:
+1.

Reply:
it can also be of type none for the optional fields , so I've added these cases